src/main/java/io/excaliburfrc/robot/commands/ShootBallsCommand.java

```diff
@@ -1,6 +1,8 @@
 package io.excaliburfrc.robot.commands;
 
-import edu.wpi.first.wpilibj2.command.*;
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
```

Comment:
There is absolutely no reason this change should be here. Spotless should keep this stuff consistent.